tests/test_single_point.py

```diff
@@ -279,20 +280,30 @@ def test_mlips(arch, device, expected_energy):
         "alignn",
         "cpu",
         -11.148092269897461,
+        "NaCl.cif",
         {"model_path": ALIGNN_PATH / "best_model.pt"},
     ),
-    ("sevennet", "cpu", -27.061979293823242, {"model_path": SEVENNET_PATH}),
-    ("sevennet", "cpu", -27.061979293823242, {}),
-    ("sevennet", "cpu", -27.061979293823242, {"model_path": "SevenNet-0_11July2024"}),
+    ("sevennet", "cpu", -27.061979293823242, "NaCl.cif", {"model_path": SEVENNET_PATH}),
+    ("sevennet", "cpu", -27.061979293823242, "NaCl.cif", {}),
+    (
+        "sevennet",
+        "cpu",
+        -27.061979293823242,
+        "NaCl.cif",
+        {"model_path": "SevenNet-0_11July2024"},
+    ),
+    ("nequip", "cpu", -169815.1282456301, "toluene.xyz", {"model_path": NEQUIP_PATH}),
```

Comment:
Out of interest, is this a remotely sensible energy?

Reply:
yes, they do not have pseudos so energy is not scaled... although I will not use that potential for anything serious.

Reply:
good thing is I have a training example for nequip... I wonder if we can use it for multi training?